.github/workflows/test.yml

```
@@ -149,7 +149,7 @@ jobs:
         continue-on-error: true
 
   github:
-    name: ${{ matrix.nvhpc && format('NVHPC {0} ({1})', matrix.nvhpc, matrix.target) || format('Github ({0}, {1}, {2}, intel={3})', matrix.os, matrix.mpi, matrix.debug, matrix.intel) }}
+    name: ${{ matrix.nvhpc && format('NVHPC {0} ({1})', matrix.nvhpc, matrix.target) || format('Github ({0}, {1}, {2}, matrix.intel && 'intel' || 'GNU')', matrix.os, matrix.mpi, matrix.debug, matrix.intel) }}
```

actionlint (1.7.12)

[error] unexpected token "IDENT" while parsing arguments of function call. expecting ",", ")"

(expression)